app/inpututils.cpp

```diff
@@ -270,6 +270,40 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
   return true;
 }
 
+QString InputUtils::renameFile( const QString &srcPath, const QString &prefix )
```

Comment:
Can we have a better name for this function? From the current name I would have thought that it takes source and destination path and just does rename without extra magic :)